src/test/unit/pages/UserProfilePage.test.tsx

```diff
@@ -0,0 +1,53 @@
+import { act, render, screen, fireEvent } from '@testing-library/react';
+import UserProfilePage from '../../../pages/UserProfilePage';
+import * as userApi from '../../../api/userApi';
+
+const mockUser = {
+    id: '123',
+    displayName: 'Mock User',
+    email: 'mock.user@example.com',
+    password: 'what is security',
+    createdDate: '2023-08-28',
+  };
+  
+const mockGetUser = jest.spyOn(userApi, 'getUser');
+
+describe('UserProfilePage', () => {
+  beforeEach(async () => {
+    await act(async () => {
+      mockGetUser.mockResolvedValue(mockUser);
+      render(<UserProfilePage id={"123"}/>);
+    });
+  });
+
+  describe('when rendered', () => {
+    it('displays user profile information', async () => {
+      const displayNameElement = await screen.findByText(mockUser.displayName);
+      const emailElement = await screen.findByText(mockUser.email);
+  
+      expect(displayNameElement).toBeInTheDocument();
+      expect(emailElement).toBeInTheDocument();
+    });
+
+    describe('when change password is initiated', () => {
+      it('changes password', () => {
+        const currentPasswordInput = screen.getByLabelText(/Current Password:/);
+        const newPasswordInput = screen.getAllByLabelText(/New Password:/)[0];
+        const confirmNewPasswordInput = screen.getByLabelText(/Confirm New Password:/);
+        const changePasswordButton = screen.getByRole('button', { name: "Change Password" });
+
+        expect(currentPasswordInput).toBeInTheDocument();
+        expect(newPasswordInput).toBeInTheDocument();
+        expect(confirmNewPasswordInput).toBeInTheDocument();
+        expect(changePasswordButton).toBeInTheDocument();
+    
+        // TODO: Test actual functionality
```

Comment:
Didn't make a ticket for this since actual authentication is a long way off, so let me know if you'd prefer I 1) go ahead and make the ticket or 2) delete the todo

Reply:
I think it's fine to delete the todo. We'll remember to do authentication when we get to it, since that is a major thing :)

Reply:
Yep sounds good! Will just delete it for now